config.js

```diff
@@ -446,6 +446,8 @@ config.tincanapi = {
  * @param  {String}     token           The mixpanel api token
  */
 config.mixpanel = {
-    'enabled': false,
-    'token': 'f3e9fce119d357b745a8dfa36248d632'
+    'enabled': true,
```

Comment:
And we should probably keep MixPanel disabled by default as well, plus we should not add real values as the `apikey` and `apisecret` for a paid service.